docs/source/savant_101/00_streaming_model.rst

```diff
@@ -23,9 +23,9 @@ The module de-multiplexes those streams internally to the form where each stream
 
 .. image:: ../_static/img/0_streaming_model_mux_demux.png
 
-The developer doesn't care about how to handle multiple streams. However, they must be aware that there are numerous streams, not one. Especially, It is important when the code maintains the state for every stream, e.g., when counting people on a per-stream basis. To ensure the state is handled properly, the developer must consider the stream's ``source_id``. The API allows the developer to retrieve per-stream information to control the situation.
+Developers does not care about how to handle multiple streams. However, they must be aware that there are numerous streams, not one. Especially, It is important when the code maintains a per-stream state, e.g., when counting people on video. To ensure the state is handled properly, developers must consider using stream ``source_id``. The API allows developers to retrieve per-stream information to control the situation.
 
-Such a multiplexed/de-multiplexed operation model is very beneficial because any well-made Savant module immediately becomes production-ready. The module doesn't know whether it processes a file-based stream, a live stream, or just a bunch of images; it doesn't distinguish between them. The framework ensures the processing for all those streams in the same way. More to say, a mix of various streams can be processed simultaneously by the same module.
+Such a multiplexed/de-multiplexed operation model is very beneficial because any well-made Savant module immediately becomes production-ready. The module does not know whether it processes a file-based stream, a live stream, or just a bunch of images; it does not distinguish between them. The framework ensures the processing for all those streams in the same way. More to say, a mix of various streams can be processed simultaneously by the same module.
```

Comment:
```suggestion
Such a multiplexed/de-multiplexed operation model is very beneficial because any well-made Savant module immediately becomes production-ready. The module does not know whether it processes a file-based stream, a live stream, or just a bunch of images; it does not distinguish between them. The framework ensures the processing for all those streams in the same way. What's more, a mix of various streams can be processed simultaneously by the same module.
```